src/main/java/frc/robot/components/VisionData.java

```diff
@@ -8,29 +8,31 @@
     public double bx;
     public double by;
     public double br;
-
     public Integer hoopStatus;
     public Integer ballStatus;
     public boolean waitForOtherRobot;
 
     public boolean isHoopValid() {
         if (hoopStatus != 1) {
             return false;
-        } else {
+        } 
+        
```

Comment:
I don't love the extra lines before the `else` clauses, but I'll leave it to the team to decide